CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 ## Changelog
 
+### 2020.04.1
+
+**Released**: 2020.04.21
+
+**Commit Delta**: [Change from 2020.03.1 release](https://github.com/plus3it/spel/compare/2020.03.1...2020.04.1)
+
+**Manifests**: <https://github.com/plus3it/spel/blob/2020.04.1/manifests>
+
+**Summary**:
+
+*   "Extra" packages updated in this release:
+    - aws-cli/1.18.42
+
+*   RHEL-only:
+    - Minor-release update to 7.8 (see [vendor release notes](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.8_release_notes/index) for additional details)
+    - rh-amazon-rhui-client-3.0.26
+
+*   Vagrant box:
+    - Updated to CentOS 7.7
```

Comment:
It looks like the centos 7.6 build was a hiccup in the March build that I missed at the time. Can you remove this here, and add a note to the prior 2020.03.1 entry indicating the discrepancy? It was 7.7 in February, the 7.6 for some reason in March...

Reply:
I moved the note to the 2020.03.1 entry.

Reply:
I need to redo it.  I'm going to change the note...

Reply:
Looks to be an outdated manifest for virtualbox in March?  Versions decremented.

Reply:
Note updated in March